source/slang/hlsl.meta.slang

```diff
@@ -1669,6 +1758,10 @@ matrix<T, N, M> max(matrix<T, N, M> x, matrix<T, N, M> y)
 
 // minimum
 __generic<T : __BuiltinArithmeticType>
+__target_intrinsic(hlsl)
+__target_intrinsic(glsl)
+__target_intrinsic(cuda, "$P_min($0, $1)")
```

Comment:
Yes, I would lump this under the "capability" feature. Ideally a `__target_intrinsic(C, D)` would have `C` be an arbitrary capability so that the modifier says "on targets that have all the capabilities in `C`, the definition of this intrinsic is `D`". If we set it up so that the `cpu` and `cuda` capabilities both "inherit" from a base capability (something like `cpu_like`?) then we could just write a single `__target_intrinsic` with the base capability.

Ideally capabilities should also support combination with `&` and `|` so that we could also write `__target_intrinsic(cpu | cuda, ...)`.